changelogs/fragments/10741-pacemaker-cluster-cleanup-deprecate.yml

```
@@ -0,0 +1,2 @@
+deprecated_features:
+  - pacemaker_cluster - the state ``cleanup`` will be deprecated in community.general 12.0.0 (https://github.com/ansible-collections/community.general/pull/10741).
```

This PR **is** doing the deprecation. You mean removal.

The removal should happen a lot later, since the alternative is just there now, and this deprecation will only be there in the next major release (12.0.0). Let's use 14.0.0, that's another year after 12.0.0 comes out:
```suggestion
  - pacemaker_cluster - the state ``cleanup`` will be removed from community.general 14.0.0 (https://github.com/ansible-collections/community.general/pull/10741).
```